### PR TITLE
modern-sysu-thesis:0.1.0

### PR DESCRIPTION
**Pull request by huangjj27**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: Explain what the package does and why it's useful.

this package is a typst-based thesis template for Sun Yat-Sen University.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by Enter-tainer** [comment]
> selected a name that isn't the most obvious or canonical name for what the package does

**Comment by Enter-tainer** [comment]
you may want to use "modern" instead of "morden"

**Comment by huangjj27** [comment]
> you may want to use "modern" instead of "morden"

You‘re right！ I mis-spelled it. QAQ

**Comment by huangjj27** [comment]
I want to make the `template` files as a git submodule (planned for future version, no influence for this PR), will it work for the current package system? #658

File: packages/preview/modern-sysu-thesis/0.1.0/layouts/doc.pdf
Comment (elegaanz) [inline packages/preview/modern-sysu-thesis/0.1.0/layouts/doc.pdf:1]: I think this file was auto-generated and should not be committed. Could you please remove it?

File: packages/preview/modern-sysu-thesis/0.1.0/pages/bachelor-cover.typ [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  // 4.  正式渲染
+
+  pagebreak(weak: true, to: if twoside { "odd" })
+
+  // 居中对齐
+  set align(center)
+
+  // 封面校徽
+  // 使用校方官方 VI 系统的 logo，来源：https://home3.sysu.edu.cn/sysuvi/index.html
+  image("../assets/vi/sysu_logo.svg", width: 3cm)
```
Comment (elegaanz) [inline packages/preview/modern-sysu-thesis/0.1.0/pages/bachelor-cover.typ:126]: The `assets` folder doesn't exist.
```suggestion
  image("../vi/sysu_logo.svg", width: 3cm)
```

**Comment by elegaanz** [comment]
Thank you!

**Pull request merged**